sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/ShowTablesSuite.scala

```diff
@@ -136,6 +136,24 @@ trait ShowTablesSuiteBase extends command.ShowTablesSuiteBase with command.Tests
     assert(msg.matches("(Database|Namespace) 'unknown' not found"))
   }
 
+  test("[SPARK-2222] keep the legacy output schema") {
```

Comment:
```suggestion
  test("SPARK-38482: keep the legacy output schema") {
```

Reply:
Oh, my bad, addressed and could you please take a look again, thanks